python/lsst/pipe/tasks/extended_psf.py

```diff
@@ -28,20 +28,61 @@
     "StackBrightStarsTask",
     "MeasureExtendedPsfConfig",
     "MeasureExtendedPsfTask",
+    "DetectorsInRegion",
 ]
 
 from dataclasses import dataclass
-from typing import List
 
 from lsst.afw.fits import Fits, readMetadata
 from lsst.afw.image import ImageF, MaskedImageF, MaskX
 from lsst.afw.math import StatisticsControl, statisticsStack, stringToStatisticsProperty
 from lsst.daf.base import PropertyList
 from lsst.geom import Extent2I
-from lsst.pex.config import ChoiceField, Config, ConfigurableField, DictField, Field, ListField
+from lsst.pex.config import ChoiceField, Config, ConfigDictField, ConfigurableField, Field, ListField
 from lsst.pipe.base import PipelineTaskConfig, PipelineTaskConnections, Struct, Task
-from lsst.pipe.tasks.coaddBase import subBBoxIter
 from lsst.pipe.base.connectionTypes import Input, Output
+from lsst.pipe.tasks.coaddBase import subBBoxIter
```

Comment:
The rest of the imports appear to be sorted as `isort` would sort them, so pushing this down by one line seems unnecessary.

Reply:
This change was made by `isort`, so I suspect this is the correct location for this import in `isort`-land.

Reply:
Oh, my bad. I misread it initially.